styles/mariokart.css

```diff
@@ -14,6 +14,20 @@
     font-family: 'MuseoModerno';
     src: url('museomoderno-bold.ttf');
 }
+div div input {
+	color: #6d9eeb;
+	background-color: #1c4587;
+	border-color: #4a6fa8;
+	border-width: 10px;
+	border-radius: 10px;
+	transition: cubic-bezier(0.215, 0.610, 0.355, 2);
+	transition-property: all;
+	transition-duration: 0.5s;
+}
+  div div input:hover {
+	transform: rotate(-5deg) scale(85%);
+	filter: brightness(200%);
+}
```

Comment:
Targetting all inputs inside div's divs seem quite impactful, have you tested it doesn't have side-effects?

Reply:
I'd suggest limiting to `button`s and `input[type="button"]` (I already do it line 503), otherwhise it also affects text inputs and radio inputs, which is kinda strange